ci/homebrew-tap/ci-deploy.sh

```diff
@@ -29,4 +30,10 @@ if [[ $do_push == true ]]; then
   git push origin master
 else
   echo "Refraining from push since it's a dry run"
+  sed -i "s/pypi.io/$pypi_host/" git-machete.rb
+  # install git-machete from local formula with homebrew
+  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)" < /dev/null
```

Comment:
Hmmm why double nesting? one level being `bash -c`, the other being `$(...)`

Reply:
hmm, this line is provided by brew, but `$(...)` is to grab the body of `install.sh` and `bash -c` to execute it.

I could split it into two lines, but i do not see much improvement here.

Reply:
Oh! okay, I see the point in that, let's leave as is